index.js

```diff
@@ -5,14 +5,38 @@ console.log("Here's your header:", header)
 
 /***** Deliverable 2 *****/
 
+header.style.color = "red";
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+//  console.log('This is what the traveler object looks like: ', traveler)
+const div = document.createElement("div")
+div.innerHTML =    `<div> 
+<img src="${traveler.photo}"
+alt="${traveler.name}">
+<h2> ${traveler.name}</h2>
+<em>${traveler.nickname}</em> </div>`
 
+const parentDiv = document.querySelector("div.traveler")
+parentDiv.prepend(div)
```

Comment:
Doing this would delete the inner div with the id='profile'. I see you commented out the HTML inside of this div but when you did your querySelector, you would be grabbing the first div (with class='traveler') but you did not replace the div with the id of 'profile' (you created a new div but did not give it the proper id). Since these HTML tags were already created, you could have just targeted each tag you wanted to change with a querySelector.  
const parentDiv = document.querySelector("div.traveler")
parentDiv.querySelector('img')
img.src = `${traveler.photo}`